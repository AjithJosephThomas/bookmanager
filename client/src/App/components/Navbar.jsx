import React from 'react';
import {Navbar as StyledComponent } from "../../style";
const Navbar = () => (
  <StyledComponent>
        <div>
            <h1>Book Manager</h1>
        </div>
        <div>
            <ul className="nav-menu">
                <li><a href="/allbooks">Main</a></li>
                <li><a href="/create-book">Create Book</a></li>
                <li><a href="/create-author">Create Author</a></li>
            </ul>
        </div>
  </StyledComponent>
)
export default Navbar;