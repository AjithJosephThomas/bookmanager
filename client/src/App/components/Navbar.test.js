import React from 'react';
import renderer from 'react-test-renderer';

import Navbar from './Navbar';

describe("Navbar component", () => {
    it('renders header navbar', () => {
        const navbar = renderer.create(<Navbar />).toJSON();
        expect(navbar).toMatchSnapshot();
    });
})