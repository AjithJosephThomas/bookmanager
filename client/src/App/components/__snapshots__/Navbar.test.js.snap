// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component renders header navbar 1`] = `
<div
  className="sc-AxjAm euKJqs"
>
  <div>
    <h1>
      Book Manager
    </h1>
  </div>
  <div>
    <ul
      className="nav-menu"
    >
      <li>
        <a
          href="/allbooks"
        >
          Main
        </a>
      </li>
      <li>
        <a
          href="/create-book"
        >
          Create Book
        </a>
      </li>
      <li>
        <a
          href="/create-author"
        >
          Create Author
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`renders header navbar 1`] = `
<div
  className="sc-AxjAm euKJqs"
>
  <div>
    <h1>
      Book Manager
    </h1>
  </div>
  <div>
    <ul
      className="nav-menu"
    >
      <li>
        <a
          href="/allbooks"
        >
          Main
        </a>
      </li>
      <li>
        <a
          href="/create-book"
        >
          Create Book
        </a>
      </li>
      <li>
        <a
          href="/create-author"
        >
          Create Author
        </a>
      </li>
    </ul>
  </div>
</div>
`;
